.builds/freebsd.yml

```diff
@@ -5,22 +5,33 @@ packages:
     - devel/gmake
     - devel/llvm
     - ftp/wget
+    - python
```

Comment:
I think it's `lang/python` instead: https://svnweb.freebsd.org/ports/head/lang/python/